src/views/Accessibility/index.tsx

```diff
@@ -93,13 +99,45 @@ const NotFound = () => (
 
 const Default = <Route path="*" key="508-not-found" component={NotFound} />;
 
+const ReportProblemLinkArea = ({ url }: { url: string }) => {
+  const { t } = useTranslation('accessibility');
+  return (
+    <div className="width-full report-problem-link-area">
+      <div className="grid-container">
+        <div className="grid-row padding-bottom-2">
+          <div className="grid-col-3">
+            <Link href={url} target="_blank" rel="noopener noreferrer">
+              {t('reportProblem')}
+            </Link>
+          </div>
+          <div className="grid-col-9" />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const A11yTeamPageTemplate = ({ children }: { children: React.ReactNode }) => {
```

Comment:
`A11yTeamPageTemplate` and `PageTemplate` look exactly the same.

Am I missing something?

Reply:
They are the same except for the url link.  I thought it was more readable to do it as separate components, but I can change it so that it's taking the url as a prop.

Reply:
Ohh I see.
Okay, this might be fine to leave as is it. The _ideal_ solution is too invasive to justify right now.

Reply:
We can pass a prop into the `PageTemplate` like `isAccessibilityTeam` and have that toggle the URLs. It's a step closer to ideal Think.